### DISCUSSION TITLE
How to highlight IP (v4 & v6) addresses

### DISCUSSION THREAD

**Discussion by ruffian** [LazyVim/LazyVim#2301] [Q&A] [answered]
Hi there!

I'm fairly new to LazyVim and I'm curious about the most effective method for highlighting IP addresses across all file types.

Thanks

**Comment by dpetka2001** [comment, marked as answer]
You can try in your `/lua/config/options.lua` the following
```lua
vim.api.nvim_set_hl(0, "IP_highlight", { fg = "red", bg = "None" })
vim.fn.matchadd("IP_highlight", "\\(\\d\\{1,3}\\.\\)\\{3}\\d\\{1,3}")  -- naive IPv4 regex
```
Fiddle around with the regex if you'd like to refine it.

**Reply by ruffian** [reply]
Thanks a lot
